hexapod_controller/include/hexapod_controller/hexapod_controller_simple.hpp

```diff
@@ -71,12 +71,12 @@ namespace hexapod_controller {
             _broken_legs = broken_legs;
         }
 
-        std::vector<int> broken_legs()
+        std::vector<int> broken_legs() const
```

Comment:
Why isn't it more like
```C++
const std::vector<double>& parameters() const;
const std::vector<int>& broken_legs() const;
```

These method declarations give read-only access to member variables, copy-less.